### ISSUE TITLE
Support alt_name:XX tag?

### ISSUE THREAD

**Issue by SiarheiFedartsou** [pelias/openstreetmap#583]
### Use-cases
There are localized `alt_name:XXX` tags (e.g. https://www.openstreetmap.org/way/31151623), which we don't import right now. Would be nice to do so.

### Attempted Solutions
https://github.com/pelias/openstreetmap/pull/582

### Proposal
Let's parse it and add as name alias for given language.

**Comment by SiarheiFedartsou** [comment]
Worth saying that all tags we have in schema in theory may be localized, should we support all of them in the same way? 

```
var OSM_NAMING_SCHEMA = {
  'name':             'default',
  'loc_name':         'default',
  'alt_name':         'default',
  'short_name':       'default',

  // note: these aliases are currently disabled because they are not being used when querying
  // 'int_name':         'international',
  // 'nat_name':         'national',
  // 'official_name':    'official',
  // 'old_name':         'old',
  // 'reg_name':         'regional',
  // 'sorting_name':     'sorting'
};
```

**Comment by missinglink** [comment]
Looking at the TagInfo stats, unfortunately I don't think this is going to be super impactful, this might be why we didn't implement back in 2018.

[2.4M Undefined](https://taginfo.openstreetmap.org/search?q=alt_name#keys)
[181000 English](https://taginfo.openstreetmap.org/search?q=alt_name:en#keys)
[22000 Mandarin](https://taginfo.openstreetmap.org/search?q=alt_name:zh#keys)
[6094 Spanish](https://taginfo.openstreetmap.org/search?q=alt_name:es#keys)
[1943 Thai](https://taginfo.openstreetmap.org/search?q=alt_name:th#keys)

Funnily enough it seems that supporting `alt_name:en` would have the biggest benefit, but after that and Mandarin the benefits diminish quickly, with Spanish (the most popular Indo-European language) only having 6094 entries in all of OSM, presumably some of those features would not be importable due to other tags.

Something to consider, not saying it's a bad idea, just wanted to better understand the value.

**Comment by SiarheiFedartsou** [comment]
@missinglink 
Yes, I agree it may be not that impactful, but it seems to be not a rocket science to implement, so may be worth trying, or may be I don't get some important detail? WDYT?
